pkg/sysinfo/sysinfo.go

```diff
@@ -25,17 +25,15 @@ func New(quiet bool) *SysInfo {
 			logrus.Warnf("%s", err)
 		}
 	} else {
-		_, err1 := ioutil.ReadFile(path.Join(cgroupMemoryMountpoint, "memory.limit_in_bytes"))
-		_, err2 := ioutil.ReadFile(path.Join(cgroupMemoryMountpoint, "memory.soft_limit_in_bytes"))
-		sysInfo.MemoryLimit = err1 == nil && err2 == nil
+		sysInfo.MemoryLimit = err == nil
```

Comment:
That's right, there are no extra kernel configs, one you mount memory cgroup, you'll have these files.